### ISSUE TITLE
add cosmology model

### ISSUE THREAD

**Issue by mi-dai** [lincc-frameworks/tdastro#11]
add cosmology model to generate observed magnitude based on given absolute magnitude

**Comment by jeremykubica** [comment]
Do we still need this feature? If so, can you provide more context on what is needed?

**Comment by mi-dai** [comment]
No. I think we more or less have it now. And more details should probably be defined within the model.
